src/main/groovy/nebula/plugin/info/reporting/InfoJarManifestPlugin.groovy

```diff
@@ -38,6 +43,39 @@ class InfoJarManifestPlugin implements Plugin<Project>, InfoReporterPlugin {
                     jarTask.manifest.attributes.putAll(attrs)
                 }
             }
+
+            project.afterEvaluate {
+                if(GradleKt.versionGreaterThan(project.gradle, "6.6-rc-1")) {
+                    configureMetaInfNormalization(project)
+                }
+            }
+
         }
     }
+
+    @CompileDynamic
+    private void configureMetaInfNormalization(Project project) {
+        InfoBrokerPluginExtension extension = project.extensions.getByName('infoBroker') as InfoBrokerPluginExtension
+        project.normalization.runtimeClasspath.metaInf(new Action<MetaInfNormalization>() {
+            @Override
+            void execute(MetaInfNormalization metaInfNormalization) {
+                if(extension.ignoreManifestForNormalization.isPresent() && extension.ignoreManifestForNormalization.get()) {
```

Comment:
I'm not sure I'd expose this here - that's something you'd want to configure directly, no?

Reply:
I was thinking on a extension so people can decide what to do (like outside of the nebula context). But yeah you are right... this is info broker specific.

Should we have a list of what to ignore first then?

I'm thinking:

`Build-Date: 2020-07-08_18:34:10` 
`Built-OS`
`Build-Number`
`Build-Job`
`Build-Id